src/components/ContentNoResult/ContentNoResult.tsx

```diff
@@ -20,13 +20,27 @@ import { AppConfig } from "@/config/types";
 
 interface ContentNoResultProps {
   councilConfig?: AppConfig["council"];
+  type?: "comments" | "applications" | undefined;
 }
 
-export const ContentNoResult = ({ councilConfig }: ContentNoResultProps) => {
+const getNoResultTitle = (type?: "comments" | "applications") => {
+  const titleMap: Record<"comments" | "applications", string> = {
+    comments: "comments",
+    applications: "applications",
+  };
+  return titleMap[type ?? "applications"];
+};
+
+export const ContentNoResult = ({
+  councilConfig,
+  type,
+}: ContentNoResultProps) => {
```

Comment:
We're reusing the same values in multiple places, it's cleaner to extract them into a shared type and easier to update in the future. Using the singular form for the type name better follows convention to keeps things consistent.



```ts

type ContentNoResultContentTypes = "comment" | "application";

interface ContentNoResultProps {
  councilConfig?: AppConfig["council"];
  type?: ContentNoResultContentTypes;
}

const getNoResultTitle = (type?: ContentNoResultContentTypes) => {
  const titleMap: Record<ContentNoResultContentTypes, string> = {
    comment: "comments",
    application: "applications",
  };
  return titleMap[type ?? "application"];
};

export const ContentNoResult = ({
  councilConfig,
  type,
}: ContentNoResultProps) => {
```